deploy/k8s/06b_csi_driver_controller.yaml

```diff
@@ -0,0 +1,212 @@
+---
+kind: Deployment
+apiVersion: apps/v1
+metadata:
+  name: csi-ovirtplugin
+  namespace: ovirt-cluster-csi-drivers
+  labels:
+    app.kubernetes.io/instance: ovirt.csi.k8s.io
+    app.kubernetes.io/part-of: csi-driver-ovirt
+    app.kubernetes.io/name: ovirt-csi-driver
+    app.kubernetes.io/component: plugin
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app.kubernetes.io/instance: ovirt.csi.k8s.io
+      app.kubernetes.io/part-of: csi-driver-ovirt
+      app.kubernetes.io/name: ovirt-csi-driver
+      app.kubernetes.io/component: plugin
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/instance: ovirt.csi.k8s.io
+        app.kubernetes.io/part-of: csi-driver-ovirt
+        app.kubernetes.io/name: ovirt-csi-driver
+        app.kubernetes.io/component: plugin
+    spec:
+      serviceAccountName: ovirt-csi-driver-controller-sa
+      initContainers:
+        - name: prepare-ovirt-config
+          env:
+            - name: OVIRT_URL
+              valueFrom:
+                secretKeyRef:
+                  name: ovirt-credentials
+                  key: ovirt_url
+            - name: OVIRT_USERNAME
+              valueFrom:
+                secretKeyRef:
+                  name: ovirt-credentials
+                  key: ovirt_username
+            - name: OVIRT_PASSWORD
+              valueFrom:
+                secretKeyRef:
+                  name: ovirt-credentials
+                  key: ovirt_password
+            - name: OVIRT_CAFILE
+              value: /tmp/config/ovirt-engine-ca.pem
+            - name: OVIRT_INSECURE
+              valueFrom:
+                secretKeyRef:
+                  name: ovirt-credentials
+                  key: ovirt_insecure
+            - name: OVIRT_CA_BUNDLE
+              valueFrom:
+                secretKeyRef:
+                  name: ovirt-credentials
+                  key: ovirt_ca_bundle
+          image: quay.io/rh_ee_mengel/ovirt-csi-driver:latest
```

Comment:
can you publish the container to quay.io/ovirt (ie https://quay.io/repository/ovirt/csi-driver) and not your personal quay account..

Reply:
Good point! I changed it to https://quay.io/repository/openshift/origin-ovirt-csi-driver